web/pdf_viewer.js

```diff
@@ -877,6 +886,17 @@ var PDFViewer = (function pdfViewer() {
     setFindController: function (findController) {
       this.findController = findController;
     },
+
+    /**
+     * Returns sizes of the pages.
+     * @returns {Array} Array of objects with width/height fields.
+     */
+    getPagesOverview: function () {
```

Comment:
Would `getPageSizes` perhaps be a better name, or do you want to keep it general to add more properties later on?